### DISCUSSION TITLE
Issues with the git clone command

### DISCUSSION THREAD

**Discussion by MidnightsCrow** [community/community#15162] [Other Feature Feedback, Questions, & Ideas]
whenever i type in "git clone https://github.com/**********/*********.git"
then type in my username
then password, it says:
"remote: Support for password authentication was removed on August 13, 2021. Please use a personal access token instead.
remote: Please see https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information.
fatal: Authentication failed for 'https://github.com/**********/*********.git/',"

what is my personal access, and how do i get it?

**Comment by bartr** [comment]
This is a good place to start: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
